apps/countries-api/pyproject.toml

```diff
@@ -1,15 +1,14 @@
 [tool.poetry]
-name = "countries-proxy"
+name = "countries-api"
 version = "0.1.0"
-description = "A proxy for the Applications team's homework assignment"
-authors = ["Immuta Applied Engineering <ae@immuta.com>"]
+description = "A countries API for Immuta's applications homework assignment"
```

Comment:
`integrations` homework assignment?

Reply:
I feel like it would make sense to keep it generic since the team is no longer called integrations and the assignment itself is to build some sort of an application.